src/index.styl

```diff
@@ -102,3 +106,42 @@ $pane-header-height = 64px
     &__subtitle
         font-size 12px
         margin-top 2px
+
+    &__overlay_custom
```

Comment:
is it needed in main source code?

Reply:
I have moved this out of the main source into its own css file.